blaze/compute/core.py

```diff
@@ -367,8 +371,14 @@ def swap_resources_into_scope(expr, scope):
 
 
 @dispatch(Expr, dict)
-def compute(expr, d, **kwargs):
-    """ Compute expression against data sources
+def compute(expr, d, return_type='native', **kwargs):
+    """ Compute expression against data sources.
+
+    Args:
+        expr: blaze expression
+        d: data source to compute expression on
+        return_type (optional): type to return data as. Defaults to 'core' which
```

Comment:
One more: we should note that the user can pass in a `type` explicitly as a third option besides the `'core'` and `'native'` string options.